src/dataExplorer/components/FluxQueryBuilder.tsx

```diff
@@ -10,7 +10,7 @@ import Schema from 'src/dataExplorer/components/Schema'
 // Styles
 import './FluxQueryBuilder.scss'
 
-const INITIAL_VERT_RESIZER_HANDLE = 0.2
+const INITIAL_VERT_RESIZER_HANDLE = 0.24
```

Comment:
There was a request (from Julia on [slack](https://influxdata.slack.com/archives/C030G5783M3/p1654096476089539?thread_ts=1654094364.248209&cid=C030G5783M3)) to increase the default width for schema browser. Since the clockface component `DraggableResizer` set default width by ratio instead of by a fix value, after calculation, Julia suggested to use 0.24

Reply:
🤔 🤔 🤔